src/components/CollapsableContainer.js

```diff
@@ -165,13 +163,13 @@ class CollapsableContainer extends Component {
               <div className="image-preview image-preview-small">
               </div>
             </div>
-            <Input value={dataBack.email} name="email" content="Email" example="Ej: sally-hill@gmail.com" type="email" />
+            <Input value={dataBack.email} name="email" content="Email" example="Ej: sally-hill@gmail.com" type="email" handleInputs = {this.props.handleInputs}/>
 
-            <Input value={dataBack.phone}  name="phone" content="Teléfono" example="Ej: 555·55·55·55" type="tel" />
+            <Input value={dataBack.phone}  name="phone" content="Teléfono" example="Ej: 555·55·55·55" type="tel" handleInputs = {this.props.handleInputs}/>
 
-            <Input value={dataBack.linkedin}  name="Linkedin" content="Linkedin" example="Ej: sally-hill" type="text" />
+            <Input value={dataBack.linkedin}  name="linkedin" content="Linkedin" example="Ej: sally-hill" type="text" handleInputs = {this.props.handleInputs} />
 
-            <Input value={dataBack.github} name="Github" content="Github" example="Ej: sally-hill" type="text" />
+            <Input value={dataBack.github} name="github" content="ithub" example="Ej: sally-hill" type="text" handleInputs = {this.props.handleInputs} />
```

Comment:
You're missing a G in `content="ithub"`!! Also, write GitHub with capital H (only in content)